libr/core/cmd.c

```diff
@@ -3999,6 +3999,10 @@ static void cmd_descriptor_init(RCore *core) {
 	}
 }
 
+static int core_cmd0_wrapper(void *core, const char *cmd) {
```

Comment:
just for the void* -> RCore* so its not really a crashy problem. i was about to do a typedef and cast instead of doing this

Reply:
I did this given what we said in the past about cast being unsafe in case something changes in the signature.. the cast will just work silently, the wrapper will give you an error back. But I'm ok with the cast too.